lib/commandsExecute.js

```diff
@@ -35,13 +35,13 @@ var commandsExecute = function(name, params, statusCallback) {
         var timeStamp = Date.now();
         if (res.headers['content-type'] === 'application/json; charset=utf-8') {
             if (res.body.state !== 'inProgress') {
-              deferred.resolve({ body: res.body, response: res.res });
+              deferred.resolve({ response: res.res });
```

Comment:
This seems backward to me. Wouldn't we want something more like `deferred.resolve(res.body)` so that the promise results in just the relevant body object data?